R/lint.R

```diff
@@ -24,7 +24,9 @@
 #' The latter (inline data) applies whenever `filename` has a newline character (\\n).
 #' @param linters a named list of linter functions to apply. See [linters] for a full list of default and available
 #' linters.
-#' @param ... additional arguments passed to [exclude()].
+#' @param ... Provide additional arguments to be passed to:
+#' - [exclude()] (in case of `lintr()`; e.g. `lints` or `exclusions`)
```

Comment:
typo?

Reply:
Sorry, I don't see what's the typo here.

Reply:
function is lint()?